alfresco-transformer-base/src/main/java/org/alfresco/transformer/AbstractTransformerController.java

```diff
@@ -148,12 +150,12 @@
         return new ResponseEntity<>(transformConfig, OK);
     }
 
```

Comment:
General note: Please try to avoid changing the current style by adding or removing spaces. If you are going to do that it needs to be in a separate PR so it is not all mixed up with functional changes.